packages/integration-sdk-runtime/src/logger/registerEventEmitters.ts

```diff
@@ -73,25 +91,27 @@ export function unregisterEventEmitters({
   uncaughtExceptionListener,
 }: RegisteredEventEmitters) {
   process.nextTick(() => {
-    process.removeListener(
-      'multipleResolves',
-      multipleResolveListener,
-    );
-    process.removeListener(
-      'unhandledRejection',
-      unhandledRejectionListener,
-    );
-    process.removeListener(
-      'uncaughtException',
-      uncaughtExceptionListener,
-    );
+    process.removeListener('multipleResolves', multipleResolveListener);
+    process.removeListener('unhandledRejection', unhandledRejectionListener);
+    process.removeListener('uncaughtException', uncaughtExceptionListener);
   });
 }
 
+interface MinimalLogger {
```

Comment:
What do you think of the name `ErrorLogger`?

Reply:
Clearer interface name. Thanks for the suggestion 👍